Formula/boost.rb

```diff
@@ -47,6 +47,7 @@ def install
       --layout=tagged-1.66
       --user-config=user-config.jam
       -sNO_LZMA=1
+      -sNO_ZSTD=1
```

Comment:
build parameter found from the following Jamfile:

* https://github.com/boostorg/iostreams/blob/703e9a077b899417ead9fb6c35db8f03b41247b6/build/Jamfile.v2#L28-L32